app/controllers/support/performance_data/schools_controller.rb

```diff
@@ -0,0 +1,38 @@
+class Support::PerformanceData::SchoolsController < Support::PerformanceDataController
+  include ActionController::Live
+
+  def index
+    response.headers['Content-Type'] = 'application/json'
+    response.stream.write '['
+
+    school_query.find_each.with_index do |school, index|
+      response.stream.write ",\n" unless index.zero?
+      response.stream.write attrs_for(school).to_json
+    end
+
+    response.stream.write ']'
+    response.stream.close
```

Comment:
Let's get this on dev to make sure PowerBI is OK with reading a chunked response. If it does have any issues, I guess the fallback alternative would be to render it to a static file every X minutes.